src/pages/Admin/Admin.tsx

```diff
@@ -56,6 +51,7 @@ import { useQuery } from "hooks/useQuery";
 import { useSelector } from "hooks/useSelector";
 import { useUser } from "hooks/useUser";
 import { useVenueId } from "hooks/useVenueId";
+import { useShowHide } from "hooks/useShowHide";
```

Comment:
Alphabetically sorted within the `hooks/*` imports:

```suggestion
import { useSelector } from "hooks/useSelector";
import { useShowHide } from "hooks/useShowHide";
import { useUser } from "hooks/useUser";
import { useVenueId } from "hooks/useVenueId";
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1380/commits/7105aeff532bea24d5bb7b8ca47f2fb586e955d7